src/parsers/babel_parser.js

```diff
@@ -76,11 +90,9 @@ export const parse = (file: string, data: ?string, options: ?parser.ParserOption
   // Pull out the name of a CallExpression (describe/it)
   // handle cases where it's a member expression (.only)
   const getNameForNode = node => {
-    if (isFunctionCall(node) && node && node.expression && node.expression.callee) {
-      const name =
-        node.expression.callee.name ||
-        node.expression.callee.object?.name ||
-        node.expression.callee.callee?.object?.name;
+    if (isFunctionCall(node) && node.expression.callee) {
+      const rootCallee = getRootCalleeOrSelf(node.expression);
+      const name = rootCallee.name || getRootObjectOrSelf(getRootTagOrSelf(rootCallee)).name;
```

Comment:
use the `deepGet` above, we can express the order and attributes in a more readable way... (I know, I know, "readability" is rather subjective...)

```suggestion
      const callee = deepGet(node.expression, 'callee');
      const name =
        callee.name ||
        deepGet(callee, 'object').name ||
        deepGet(callee, 'tag').name ||
        deepGet(callee, 'tag', 'object').name;
```

Reply:
> use the `deepGet` above, we can express the order and attributes in a more readable way... (I know, I know, "readability" is rather subjective...)

I've added this, except I did not include `deepGet(callee, 'tag').name`, as that's not a valid case (`describe` or `it` themselves are never a tag).

Reply:
good point